packages/cloudformation/templates/givesource-events.yml

```diff
@@ -1,68 +0,0 @@
----
-AWSTemplateFormatVersion: 2010-09-09
```

Comment:
This file didn't appear to be used.  The resources it defines also exist in givesource-custom.yml.